ci-operator/config/openshift-metal3/terraform-provider-ironic/openshift-metal3-terraform-provider-ironic-master.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  base:
+    name: "4.10"
```

Comment:
Probably now that we are the past the code freeze, all the 4.10 references could be directly updated to 4.11?